### PR TITLE
Homework 05

### PR DESCRIPTION
**Pull request by jurisk**
Also deployed to https://greatest-afternoon.surge.sh/

### CODE REVIEW

File: homeworks/05-redux-basic/001-combineReducer.md [hunk 1]
```diff
@@ -14,4 +14,4 @@ const appStore = createStore(app);
 ```
 
 Expected: 
-The reducer counter2 should have diffenent actions type name and they should be dispatched after counter's actions.
+The reducer counter2 should have different actions type name, and they should be dispatched after counter's actions.
```
Comment (avilde) [inline homeworks/05-redux-basic/001-combineReducer.md:17]: Good job on updating typos in task description 👍🏻

File: homeworks/05-redux-basic/package.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "05-redux-basic",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
```
Comment (avilde) [inline homeworks/05-redux-basic/package.json:6]: Not sure why Create React App haven't fixed this yet in their code but testing libraries and types should be in `devDependencies`.

File: homeworks/05-redux-basic/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (avilde) [inline homeworks/05-redux-basic/public/index.html:27]: Not really part of task but you can give a title of your app )

File: homeworks/05-redux-basic/src/combine-reducers.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+type Reducer<S, A> = (state: S | undefined, action: A) => S
+type ReducerMap<S, A> = {
+    [K in keyof S]: Reducer<S[K], A>
+}
+
+export const combineReducers = <S, A>(
```
Comment (avilde) [inline homeworks/05-redux-basic/src/combine-reducers.ts:6]: Not quite sure why this is needed - Redux library already has this function `combineReducers`.

**Review by avilde — APPROVED** [review]
## Review

# Good parts
* ui created to test out application code
* clean code
* task is done (reducers combined)

# What could be improved
* Jest unit tests for reducers would have been great addition

# Summary
Good job with creating UI, Juris! 👍🏻

File: homeworks/05-redux-basic/src/combine-reducers.ts (hunk 4 above)
Comment (evo-home-task) [reply homeworks/05-redux-basic/src/combine-reducers.ts:6]: This was the other homework. I just did it together to make sure my `combineReducers` worked as intended.

**Pull request merged**